docs/graph/providers/InteractiveProviderBehavior.md

```diff
@@ -1,58 +0,0 @@
----
```

Comment:
We don't have the setup to remove/redirect docs still. We should put a header and just move the link in the TOC for this to the archive.

Reply:
Do what with the header?

Reply:
Like what we do on all the deprecated pages and like how you updated here:
https://github.com/MicrosoftDocs/WindowsCommunityToolkitDocs/pull/520/files#diff-7cc5053fd55d09488f32ecbacceb529effb5a85cb827cc98df1eb15e149dea7eR13-R15